homeassistant/components/mobile_app/webhook.py

```diff
@@ -538,3 +543,35 @@ async def webhook_get_config(hass, config_entry, data):
         pass
 
     return webhook_response(resp, registration=config_entry.data)
+
+
+@WEBHOOK_COMMANDS.register("get_entities_by_area")
+async def webhook_get_entities_by_area(hass, config_entry, data):
+    """Handle a get entities grouped by area webhook."""
+    area_to_entity_map = {}
+    resp = []
+
+    area_registry = await ar.async_get_registry(hass)
+    device_registry = await dr.async_get_registry(hass)
+    entity_registry = await er.async_get_registry(hass)
+
+    entities = entity_registry.entities
```

Comment:
I'd rename `entities` to eg `entity_entries` as they are not `Entity` instances but `RegistryEntry` instances.